egs2/chime7_task1/diar_asr1/README.md

```diff
@@ -0,0 +1,188 @@
+Mixer 6 performance. <br>
+
+
+
+### Reproducing the Baseline Results.
+
+#### Training the ASR model
+
+Follow the instructions in the acoustic sub-track recipe `egs2/chime7_task1/README.md`. <br>
+Then you can create a symbolic link to your trained ASR model (which will be in the `../asr1/exp` folder). <br>
+For example, to create a symbolic link to your trained ASR model in the exp folder here in `diar_asr1`:
+```bash
+mkdir exp # if it does not exists yet in this folder it should be created
+ln -s ../asr1/exp/MY_ASR_SYSTEM ./exp
+```
+
+**Important** <br>
+The recipe stage 0 here will re-create the data. If you want to avoid
+this and use the data already created in the acoustic sub-track recipe `egs2/chime7_task1/README.md`
+you can also create a symbolic link to that one:
+```bash
+ln -s ../asr1/chime7_task1 .
+```
+
+#### Main Track with Pyannote-based Diarization System
+To reproduce our results which use our pre-trained ASR model [https://huggingface.co/popcornell/chime7_task1_asr1_baseline](https://huggingface.co/popcornell/chime7_task1_asr1_baseline) and pre-trained
+[pyannote segmentation model](https://huggingface.co/popcornell/pyannote-segmentation-chime6-mixer6)
+you can run:
+```bash
+./run.sh --chime7-root YOUR_PATH_TO_CHiME7_ROOT --stage 2 --ngpu YOUR_NUMBER_OF_GPUs \
+--use-pretrained popcornell/chime7_task1_asr1_baseline \
+--decode-only 1 --gss-max-batch-dur 30-360-DEPENDING_ON_GPU_MEM \
+--pyan-use-pretrained popcornell/pyannote-segmentation-chime6-mixer6
+```
+You can also play with diarization hyperparameters such as:
+1. `--diar-merge-closer`: merge segments after diarization from same speaker that are closer than this value.
+2. `--diar-max-length-merged`: max length allowed for segments that are merged (default 60, you may want to lower this due to GSS memory consumption).
+3. `--pyannote-max_batch_size`: max batch size used in inference when extracting embeddings (increase to speed up, default 32).
+
+as said merge-closer can have quite an impact on the final WER.
+
+---
+If you want to run this recipe from scratch, **including dataset generation** and pyannote segmentation
+model fine-tuning you can run it from stage 0:
+```bash
+./run.sh --chime6-root YOUR_PATH_TO_CHiME6 --dipco-root PATH_WHERE_DOWNLOAD_DIPCO \
+--mixer6-root YOUR_PATH_TO_MIXER6 --stage 0 --ngpu YOUR_NUMBER_OF_GPUs \
+--use-pretrained popcornell/chime7_task1_asr1_baseline \
+--decode-only 1 --gss-max-batch-dur 30-360-DEPENDING_ON_GPU_MEM \
+--pyan-use-pretrained popcornell/pyannote-segmentation-chime6-mixer6
+```
+---
+**If you want only to generate data you can run only stage 0.**
+```bash
+./run.sh --chime6-root YOUR_PATH_TO_CHiME6 --dipco-root PATH_WHERE_DOWNLOAD_DIPCO \
+--mixer6-root YOUR_PATH_TO_MIXER6 --stage 0 --stop-stage 0
+```
+Please refer to `egs2/chime7_task1/README.md` for additional infos such as
```

Comment:
From my environment, this part is highlighted. I don't know why.

<img width="1026" alt="image" src="https://user-images.githubusercontent.com/11741550/230236546-3d3b9a9b-2073-47b9-9967-4db383682992.png">

Reply:
I fixed this